el_gato/el_gato.py

```diff
@@ -1242,14 +1242,11 @@ def process_reads(contig_dict: dict, read_info_dict: dict, ref: Ref, outdir: str
         a = Allele()
         a.allele_id = '-'
         alleles['neuA_neuAH'] = [a]
-        neuA_catch = ''
-
-    if len([i for i in ref.REF_POSITIONS.keys() if 'neuA' in i]) > 1:
-        cov_sorted = sorted(
```

Comment:
deleting the assignment of the cov_sorted variable here will cause a runtime error if there is ever more than one neuA reference that meets the coverage threshold of 99% horizontal coverage.